includes/class-sensei-core-lesson-modules.php

```diff
@@ -1,6 +1,7 @@
 <?php
 
-if ( ! defined( 'ABSPATH' ) ) { exit; }
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; }
```

Comment:
This is weird...

Reply:
I've seen this on other phpcfb. Sometime running it a second time fixes it, but other times I run a find-and-replace for this.

Reply:
Yeah, spot-checked a couple other files and it didn't do that. 🤷‍♀️

Reply:
I'm not going to worry about it since it shouldn't break anything.